src/main/java/games/strategy/persistence/memento/serializable/SerializableMementoExportException.java

```diff
@@ -0,0 +1,49 @@
+package games.strategy.persistence.memento.serializable;
+
+/**
+ * A checked exception that indicates an error occurred while exporting a memento in Java object serialization format.
+ *
+ * <p>
+ * Instances of this class are thread safe.
+ * </p>
+ */
+public final class SerializableMementoExportException extends Exception {
+  private static final long serialVersionUID = -4053920530609554772L;
+
+  /**
+   * Initializes a new instance of the {@code SerializableMementoExportException} class with no detail message and no
+   * cause.
+   */
+  public SerializableMementoExportException() {}
```

Comment:
Indeed, though it appears we only use one out of four constructors in this class: https://github.com/triplea-game/triplea/pull/1889/files/3fadd15e35527fe22e12f696c117229876cb7444#diff-de7ed79bf891420efd901fc312df1469R52

I would recommend removing the unused constructors, would be less to read/understand and less commentary to keep in sync in case this code is ever updated here.